frontend/src/app/data-entry/gift-form/gift-categories-form/gift-categories-form.component.ts

```diff
@@ -0,0 +1,54 @@
+import { Component, DestroyRef } from "@angular/core";
```

Comment:
Just a few imports/properties we might need in the future. These are not used yet.